src/Eloquent/HybridRelations.php

```diff
@@ -328,6 +333,125 @@ public function belongsToMany(
         );
     }
 
+    /**
+     * Define a morph-to-many relationship.
+     *
+     * @param  string $related
+     * @param    string $name
+     * @param  null   $table
+     * @param  null   $foreignPivotKey
+     * @param  null   $relatedPivotKey
+     * @param  null   $parentKey
+     * @param  null   $relatedKey
+     * @param  null   $relation
+     * @param  bool   $inverse
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\MorphToMany
+     */
+    public function morphToMany(
+        $related,
+        $name,
+        $table = null,
+        $foreignPivotKey = null,
+        $relatedPivotKey = null,
+        $parentKey = null,
+        $relatedKey = null,
+        $relation = null,
+        $inverse = false,
+    ) {
+        // If no relationship name was passed, we will pull backtraces to get the
+        // name of the calling function. We will use that function name as the
+        // title of this relation since that is a great convention to apply.
+        if ($relation === null) {
+            $relation = $this->guessBelongsToManyRelation();
+        }
+
+        // Check if it is a relation with an original model.
+        if (! is_subclass_of($related, Model::class)) {
+            return parent::MorphToMany(
```

Comment:
Should this be `morphToMany` (with a minuscule "m")? I know method calls aren't case-sensitive in PHP, but still 😉 
```suggestion
            return parent::morphToMany(
```